src/lib/components/forms/FormSubmitButton.svelte

```diff
@@ -10,12 +10,14 @@
   export let superform: SuperForm<T>;
   let clazz: string | undefined = undefined;
   export { clazz as class };
+  export let form: string | undefined = undefined;
   const { delayed } = superform;
 </script>
 
 <LoadingButton
   class={clazz}
   type="submit"
+  {form}
```

Comment:
What does this do?

Reply:
![image](https://github.com/user-attachments/assets/9455ecd5-3936-4692-a49f-b70c57be2887)